paddle/fluid/framework/fleet/fleet_wrapper.cc

```diff
@@ -320,13 +368,16 @@ void FleetWrapper::PushSparseVarsWithLabelAsync(
     std::vector<std::vector<float>>* push_values,
     std::vector<::std::future<int32_t>>* push_sparse_status,
     const int batch_size, const bool use_cvm, const bool dump_slot,
-    std::vector<uint64_t>* sparse_push_keys, const bool no_cvm) {
+    std::vector<uint64_t>* sparse_push_keys, const bool no_cvm,
+    const paddle::platform::Place& place,
+    std::vector<float>& sparse_grad_region) {
 #ifdef PADDLE_WITH_PSLIB
   int offset = 2;
   int slot_offset = 0;
   int grad_dim = emb_dim;
   int show_index = 0;
   int click_index = 1;
+  bool in_cpu = true;
```

Comment:
is in_cpu necessary for this part? how about we use place to verify the devices we are running